plugin/applescript/update.scpt

```diff
@@ -3,11 +3,12 @@ tell application "System Events"
 end tell
 
 on update_xmark()
-    tell application "{{ app }}"
+    tell application "{{ app_loc }}"
         set u to "file://{{ outurl }}"
         repeat with w in every window
             set tab_index to 0
-            repeat with t in every tab of w
+            set tablist to tabs of w
+            repeat with t in tablist
```

Comment:
Aha, so you mean, using `s:app_loc` instead `s:app` is not sufficient to fix the problem? (sorry I can't reproduce the problem, so I'm just guessing)